app/domain/UserInfo.scala

```diff
@@ -38,8 +48,24 @@ case class UserInfo(given_name: Option[String] = None,
                     group_profile_url: Option[String] = None
                    )
 
+object UserInfo {
```

Comment:
As per our discussion, this will not change the existing responses to the vendor in any way.